plugin/perf.go

```diff
@@ -62,3 +64,26 @@ func (pp *PerfPlugin) GetPanelConfig() (*PanelConfig, error) {
 		},
 	}, nil
 }
+
+func isPerfEnabled(projectRef model.ProjectRef, projects []string) bool {
+	flags, err := evergreen.GetServiceFlags()
+	if err != nil {
+		return false
+	}
+	if flags.PluginAdminPageDisabled {
+		lastGoodVersion, err := model.FindVersionByLastKnownGoodConfig(projectRef.Id, -1)
+		if err == nil && lastGoodVersion != nil {
+			parserProject, err := model.ParserProjectFindOneById(lastGoodVersion.Id)
+			if err == nil && parserProject != nil {
```

Comment:
I believe we only want to return early if parserProject.PerfEnabled is specified, right? same to the one that's called in the task resolver.
is there a way to make this a helper function that could be used both here and in the resolver?